boards/x86/qemu_x86/qemu_x86_64_nokpti.yaml

```diff
@@ -0,0 +1,13 @@
+identifier: qemu_x86_64_nokpti
+name: QEMU Emulation for X86_64 (KPTI disabled)
+type: qemu
+arch: x86
+toolchain:
+  - zephyr
+  - xtools
+simulation: qemu
+testing:
+  default: true
+  ignore_tags:
```

Comment:
get rid of this `ignore_tags` and use the same `only_tags` in the 32-bit version.

that ensures this build target will only be used in CI by tests that are tagged with `kernel` and `userspace`